packages/externals/create-electron-app/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "create-electron-app",
+  "version": "1.0.0",
```

Comment:
I think you'll need to exclude the `externals` folder the `bump` and `publish` scripts?  Or maybe it's ok that `create-electron-app` bumps in sync with the rest of `@electron-forge/*`